test/gtest/ucs/test_mpool.cc

```diff
@@ -14,6 +14,16 @@ extern "C" {
 #include <queue>
 
 class test_mpool : public ucs::test {
+public:
+    typedef struct {
+        const size_t                elem_size;
```

Comment:
Maybe use `constexpr` where appropriate?

Reply:
This is a type, not function nor variable, so constexpr does not apply to it?